utilities/build_julia.sh

```diff
@@ -66,6 +66,9 @@ echo "--- Quick consistency checks"
 ${JULIA_EXE} -e "import Test; Test.@test Sys.ARCH == :${ARCH:?}"
 ${JULIA_EXE} -e "import Test; Test.@test Sys.WORD_SIZE == ${EXPECTED_WORD_SIZE:?}"
 
+echo "--- Show build stats"
+${MAKE} build-stats
```

Comment:
```suggestion
${MAKE} "${MFLAGS[@]}" build-stats
```

Reply:
bump

Reply:
Personally I don't understand this change but perhaps you could ping someone that might? Or explain?

Reply:
It adds the various make options that are already used when building julia, defined here https://github.com/IanButterworth/julia-buildkite/blob/9fcbe0713621dfec83f0c621c1c5fa3fca5ea82f/utilities/build_julia.sh#L23-L45.

This fixes the current error with assert builds as it will now build with the `MAKE_FLAGS` defined in the .arches file, i.e. it includes `FORCE_ASSERTIONS=1,LLVM_ASSERTIONS=1` in the make flags.

Let me know if that's not clear.

Reply:
Ah ok. Thanks